datafusion/sqllogictest/test_files/struct.slt

```diff
@@ -492,9 +492,18 @@ Struct("r": Utf8, "c": Float64)
 statement ok
 drop table t;
 
-query error DataFusion error: Optimizer rule 'simplify_expressions' failed[\s\S]*Arrow error: Cast error: Cannot cast string 'a' to value of Float64 type
+statement ok
```

Comment:
These tests now work thanks to @brancz in 
- https://github.com/apache/arrow-rs/pull/8871